daemons/based/based_callbacks.c

```diff
@@ -1069,6 +1107,11 @@ cib_process_request(xmlNode *request, gboolean privileged,
         return rc;
     }
 
+    if (pcmk_is_set(call_options, cib_transaction)) {
+        CRM_CHECK(pcmk_is_set(operation->flags, cib_op_attr_transaction),
+                  return -EOPNOTSUPP);
```

Comment:
Probably not a CHECK ... it's purely incorrect usage by client, right?

Reply:
No, this is a server issue. The server (`based_extend_transaction()`) should have rejected it at extend time. If we reach this point, it means we're in the process of committing a transaction. So we allowed an unsupported request through and we have a bug.

I'll add a comment.